### DISCUSSION TITLE
Please specify `upload_port` error when trying to flash

### DISCUSSION THREAD

**Discussion by NExZtClan** [ExpressLRS/ExpressLRS#1236] [Q&A]
I have a betafpv 915mhz module and when trying to flash i keep getting "Please specify `upload_port`". I've already tried holding down the button on the inside but that doesn't work. Please help.

**Comment by brandonrc** [comment]
Which tx exactly? Betafpv makes a few 915

Also is this on windows 10? 

Can you open device manager and see a com device when you plug the USB in.

**Reply by NExZtClan** [reply]
I figured it out. i just forgot to install the driver. thanks for trying to help.
